conans/client/build/autotools_environment.py

```diff
@@ -104,21 +112,28 @@ def _get_triplet(self, the_arch, the_os):
 
         return "%s-%s" % (machine, op_system)
 
-    def _get_host_build_target_flags(self, arch_detected, os_detected):
+    def _get_host_build_target_flags(self, arch_detected=None, os_detected=None):
```

Comment:
The optional parameters can be removed, it is a private method and the only caller is not using them. Right?

Reply:
Yes, I left the parameters in place and defaulted because it is unit tested somewhere using custom parameters. I'll have a look to modify the test